runelite-client/src/main/java/net/runelite/client/plugins/osrswikia/OSRSWikiaPlugin.java

```diff
@@ -0,0 +1,66 @@
+package net.runelite.client.plugins.osrswikia;
+
+import com.google.common.eventbus.Subscribe;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import net.runelite.api.events.CommandExecuted;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.util.LinkBrowser;
+
+@PluginDescriptor(
+	name = "OSRS Wikia",
+	enabledByDefault = false
+)
+public class OSRSWikiaPlugin extends Plugin
+{
+	private final String BASEURI = "http://oldschoolrunescape.wikia.com/wiki";
```

Comment:
Is there https:// variant? If yes, you should use that one instead.

Reply:
Doesn't appear so but can't hurt to include it. Amended

Reply:
HTTPS redirects back to HTTP? :sob: